config/initializers/new_framework_defaults.rb

```diff
@@ -19,4 +19,4 @@
 ActiveSupport.to_time_preserves_timezone = false
 
 # Require `belongs_to` associations by default. Previous versions had false.
-Rails.application.config.active_record.belongs_to_required_by_default = false
+Rails.application.config.active_record.belongs_to_required_by_default = true
```

Comment:
This line and the comment above (and the empty line above it) should be removed – it defaults to `true` and this is just a temporary upgrade help file, we want to get to removing the TODOs from the list.